netlify.toml

```diff
@@ -3,9 +3,11 @@
 
 [context.production.environment]
   HUGO_VERSION = "0.74.3"
+  HUGO_ENV = "production"
 
 [context.deploy-preview.environment]
   HUGO_VERSION = "0.74.3"
+  HUGO_ENV = "production"
```

Comment:
I guess this one is specifically one we don't want long term but good with it regardless.